addons/website_slides/models/res_config_settings.py

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = "res.config.settings"
 
     website_slide_google_app_key = fields.Char(related='website_id.website_slide_google_app_key', readonly=False)
+    module_website_sale = fields.Boolean("Sell courses")
```

Comment:
Put directly the bridge module `website_sale_slides`

Reply:
Done.